docs/WritingTests.asciidoc

```diff
@@ -1267,23 +1267,61 @@ clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 TESTDEBUG=1
 === Defining a custom test schedule or custom test modules
 
 Normally the test schedule, that is which test modules should be executed and
-which order, is prescribed by the +main.pm+ file within the test distribution.
-Additionally it is possible to exclude certain test modules from execution
-using the os-autoinst test variables +INCLUDE_MODULES+ and +EXCLUDE_MODULES+ as
-well as define a custom schedule using the test variable +SCHEDULE+.
-Also test modules can be defined and overridden on-the-fly using a
-downloadable asset.
+in which order, is prescribed by the +main.pm+ file within the test
+distribution.
+However, test developers can manipulate the schedule when cloning a job using some settings.
```

Comment:
I recommend to keep the older part as introduction for the different approaches, e.g. mentioning the test variables, but then add your examples to explain the specific methods in detail.

Reply:
Done